From 9c4ad75f17788a64c1e37d0b9e19ca157e01c80a Mon Sep 17 00:00:00 2001
From: Christian Marangi <ansuelsmth@gmail.com>
Date: Mon, 9 Dec 2024 18:19:06 +0100
Subject: [PATCH] net: ethernet: qualcomm: ppe: Fix unmet dependency with
 QCOM_PPE

Fix unmet dependency with QCOM_PPE on selecting SFP.

WARNING: unmet direct dependencies detected for SFP
  Depends on [m]: NETDEVICES [=y] && PHYLIB [=y] && I2C [=y] && PHYLINK [=y] && (HWMON [=m] || HWMON [=m]=n [=n])
  Selected by [y]:
  - QCOM_PPE [=y] && NETDEVICES [=y] && ETHERNET [=y] && NET_VENDOR_QUALCOMM [=y] && HAS_IOMEM [=y] && OF [=y] && COMMON_CLK [=y]

This permit correct compilation of the modules with SFP enabled.

Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>
---
 drivers/net/ethernet/qualcomm/Kconfig | 1 -
 1 file changed, 1 deletion(-)

--- a/drivers/net/ethernet/qualcomm/Kconfig
+++ b/drivers/net/ethernet/qualcomm/Kconfig
@@ -68,7 +68,6 @@ config QCOM_PPE
 	select REGMAP_MMIO
 	select PHYLINK
 	select PCS_QCOM_IPQ_UNIPHY
-	select SFP
 	help
 	  This driver supports the Qualcomm Technologies, Inc. packet
 	  process engine (PPE) available with IPQ SoC. The PPE includes
